.circleci/integration-tests/master_dag.py

```diff
@@ -169,6 +170,12 @@ def prepare_dag_dependency(task_info, execution_time):
         do_xcom_push=True,
     )
 
+    get_astronomer_providers_version = BashOperator(
+        task_id="get_astronomer_providers_version",
+        bash_command="airflow providers list -o json |  jq '.[] | select(.package_name == \"astronomer-providers\") | .version'",
```

Comment:
maybe you like to use get https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#get_repeat2

Reply:
thanks @pankajastro